vars/mavenNode.groovy

```diff
@@ -5,6 +5,19 @@ def call(Map parameters = [:], body) {
     def defaultLabel = buildId('maven')
     def label = parameters.get('label', defaultLabel)
 
+    def file = new File("/var/lib/jenkins/nodes")
+    
+    if(!file.exists()){
+        error "file or directory /var/lib/jenkins/nodes doesnot exist"
+    } else {
+        if(!file.isDirectory()){
+            error "/var/lib/jenkins/nodes exists, but it is not a directory"
```

Comment:
the message give very little clue why the build is failing because nodes is not a directory.